src/pds_doi_service/api/swagger/swagger.yaml

```diff
@@ -1,472 +1,417 @@
 openapi: 3.0.0
+
 info:
   title: Planetary Data System DOI Service API
   description: PDS API for managing DOI registration with a DOI service provider (OSTI, DataCite, etc.).
-  version: "0.2"
+  version: '0.2'
+
+# Added by API Auto Mocking Plugin
 servers:
-- url: http://localhost:8080/PDS_APIs/pds_doi_api/0.2
-  description: Local host
-- url: https://virtserver.swaggerhub.com/PDS_APIs/pds_doi_api/0.2
-  description: SwaggerHub API Auto Mocking
+  - description: Local host
+    url : http://localhost:8080/PDS_APIs/pds_doi_api/0.2
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/PDS_APIs/pds_doi_api/0.2
+
 tags:
-- name: dois
-  description: PDS DOI Core function restFull API
+  - name: dois
+    description: PDS DOI Core function restFull API
```

Comment:
@tloubrieu-jpl super minor nitpick, but I think this should be [RESTful](https://codewords.recurse.com/issues/five/what-restful-actually-means)?

Reply:
Yeah well I think it should be `ReST-ful` since:

- `ReST` = a combined abbreviation+initialism for **Re**presentational **S**tate **T**ransfer
- Hyphenated `-ful` since we're applying an English adjectival form to technical jargon

![tyler-hynes-hynies](https://user-images.githubusercontent.com/814813/155036831-010cdcfa-a41f-49b1-a8a7-259fda003ae7.gif)

Yes, it's a 3-day weekend and I'm checking my GitHub notifications! 😝